scripts/extract_abi.js

```diff
@@ -46,7 +45,7 @@ function compileFile (contractFile, c) {
       const subPath = parts.length === 0 ? '' : '/' + parts.join('/')
       let realPath = contractsFolder + subPath + '/' + path
       if (!fs.existsSync(realPath)) {
-        realPath = projectFolder + '/node_modules/' + path
+        realPath = 'node_modules/' + path
```

Comment:
because if the projectFolder was '', it would look for '/node_modules/' and fail. I guess '.' would work but the way we use 'extract_abi' this is not necessary. You are not supposed to run it manually outside of ```make server``` I think.